src/pages/my-decade-in-review/index.es.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Mi década en retrospectiva
+date: '2020-01-01'
+spoiler: Una reflexión personal.
+---
+
+Empecé esta década como un estudiante de primer año salido de la prepa. Tenía 17, no tenía un trabajo, no tenía conexiones en la industria, y no sabía un carajo. Y ahora estás leyendo mi blog! Hubiera estado orgulloso.
```

Comment:
Maybe it would be better`preparatoria` o `secundaria` o `colegio` instead of `prepa` but this might be because of the differences between how we speak Spanish in Latin America.